### PR TITLE
Multistage fitting

### PR DESCRIPTION
**Pull request by annecori**
necessary changes to spimalot for: https://github.com/ncov-ic/ncov-outputs/pull/2023

### CODE REVIEW

File: R/simulate.R [hunk 1]
```diff
@@ -61,15 +60,15 @@ spim_simulate_prepare <- function(combined, simulate_parameters, n_par,
     pars_mcmc, combined$transform)
 
   if (inflate_strain) {
-    message("Inflating strains")
+    message("Inflating strains. NOT SURE IF THIS WORKS WITH MULTISTAGE FITTING")
```
Comment (richfitz) [inline R/simulate.R:63]: No need for this comment - we should remove both inflations; I've added this issue for this: https://github.com/mrc-ide/spimalot/issues/153

```suggestion
    message("Inflating strains")
```

File: R/simulate.R [hunk 2]
```diff
@@ -61,15 +60,15 @@ spim_simulate_prepare <- function(combined, simulate_parameters, n_par,
...
     tmp <- simulate_prepare_inflate_strain(pars, state, info)
     pars <- tmp$pars
     state <- tmp$state
     info <- tmp$info
   }
 
   if (inflate_booster) {
-    message("Inflating vaccination classes")
+    message("Inflating vaccination classes. NOT SURE IF THIS WORKS WITH MULTISTAGE FITTING")
```
Comment (richfitz) [inline R/simulate.R:71]:
```suggestion
    message("Inflating vaccination classes")
```

as above

File: R/combined.R [hunk 3]
```diff
@@ -350,7 +350,7 @@ spim_population <- function(combined, ignore_uk = FALSE, by_age = TRUE,
 ##' @export
 spim_prop_infected <- function(combined, population,
                               regions = sircovid::regions("england")) {
-  idx_cum_infections <- combined$info[[1]]$info$index$cum_infections
+  idx_cum_infections <- combined$info[[1]]$info[[length(combined$info[[1]]$info)]]$index$cum_infections
```
Comment (richfitz) [inline R/combined.R:353]: Sorry, this is awful, but probably the right thing to do at the moment. Probably this needs to be looked at more generally later (e.g., only saving the last epoch's data into the fit?)

**Review by richfitz — CHANGES_REQUESTED** [review]
Couple of small things, plus please bump the version number to pass the version check

File: R/simulate.R (hunk 1 above)
Comment (annecori) [reply R/simulate.R]: done (just removed from this function)

File: R/simulate.R (hunk 2 above)
Comment (annecori) [reply R/simulate.R]: done

**Review by richfitz — APPROVED** [review]

**Pull request merged**